src/github.rs

```diff
@@ -1046,42 +1045,94 @@ pub struct Query<'a> {
     pub exclude_labels: Vec<&'a str>,
 }
 
+fn quote_reply(markdown: &str) -> String {
+    if markdown.is_empty() {
+        String::from("*No content*")
+    } else {
+        format!("\n\t> {}", markdown.replace("\n", "\n\t> "))
+    }
+}
+
 #[async_trait]
 impl<'q> IssuesQuery for Query<'q> {
     async fn query<'a>(
         &'a self,
         repo: &'a Repository,
+        include_fcp_details: bool,
         client: &'a GithubClient,
     ) -> anyhow::Result<Vec<crate::actions::IssueDecorator>> {
         let issues = repo
             .get_issues(&client, self)
             .await
             .with_context(|| "Unable to get issues.")?;
-
         let issues_decorator: Vec<_> = issues
-            .iter()
-            .map(|issue| crate::actions::IssueDecorator {
-                title: issue.title.clone(),
-                number: issue.number,
-                html_url: issue.html_url.clone(),
-                repo_name: repo.name().to_owned(),
-                labels: issue
-                    .labels
-                    .iter()
-                    .map(|l| l.name.as_ref())
-                    .collect::<Vec<_>>()
-                    .join(", "),
-                assignees: issue
-                    .assignees
-                    .iter()
-                    .map(|u| u.login.as_ref())
-                    .collect::<Vec<_>>()
-                    .join(", "),
-                updated_at: crate::actions::to_human(issue.updated_at),
+            .into_iter()
```

Comment:
may not need `into_iter` after all, initially tried this to leave the call to `rfcbot::get_all_fcps` outside of the async move block